app.py

```diff
@@ -25,6 +27,26 @@ def load_cfg():
     return cfg
 
 
+def add_gpg_key(gpg_key):
```

Comment:
What will going on if the key is already in the keyring? Whether the command will return zero exit code and will print the expected string? Whether the old or the new key will be in effect?

Reply:
Yep, the command will return zero exit code and will print the expected string.

Reply:
> Whether the old or the new key will be in effect?

How I can create a new key with the old ID?

Reply:
It seems, I meant some process of updating an expired key (with keeping the ID). AFAIS, our key have no expiration date (I don't see anything like this in pgpdump output). Maybe those keys have no expiration dates at all: I don't know. Anyway, it does not look relevant for us.